src/utilities/codap-utils.ts

```diff
@@ -10,17 +10,13 @@ import { CodapV2DataSetImporter } from "../codap/v2/codap-v2-data-set-importer";
 import { toV3CaseId } from "../codap/utilities/codap-utils";
 import { ICaseCreation } from "../codap/models/data/data-set-types";
 
-import { codapData, getDate, ICase } from "../models/codap-data";
-import { graph } from "../models/graph";
-import { ui } from "../models/ui";
-import { kInitialDimensions, kPluginName, kVersion } from "./constants";
+import { codapData } from "../models/codap-data";
 import { DstContainer, dstContainer } from "../models/dst-container";
+import { ui } from "../models/ui";
+import { kCollectionName, kDataContextName, kInitialDimensions, kPluginName, kVersion } from "./constants";
```

Comment:
A problem with putting `kDataContextName` in a different file is that it makes it harder to switch between datasets. That shouldn't hold up the PR, but it might be worth switching it back. Or if I find the time I'll try using dynamic imports so we don't have to specify the dataset name twice.